sdks/docker-enclave/default.nix

```diff
@@ -10,6 +10,7 @@
   vet,
   kernels,
   compose ? ./. + "/docker-compose.yml",
+  dockerImagesDir ? ./. + "/docker-images",
```

Comment:
I think make it list of paths, easier to handle once we have a cli tool wrapper

Reply:
done.